maintenance_plan_meter_mode/views/maintenance_views.xml

```diff
@@ -0,0 +1,60 @@
+<odoo>
+<data>
+    <record id="maintenance_plan_view_tree" model="ir.ui.view">
+        <field name="name">maintenance.plan.tree</field>
+        <field name="model">maintenance.plan</field>
+        <field name="inherit_id" ref="maintenance_plan.maintenance_plan_view_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='active']" position="before">
+                <field name="meter_mode"/>
+                <field name="meter_current_value"/>
+                <field name="meter_autoinc_maxvalue"/>
+                <field name="meter_autoinc_lastvalue"/>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="maintenance_plan_view_form" model="ir.ui.view">
+        <field name="model">maintenance.plan</field>
+        <field name="inherit_id" ref="maintenance_plan.maintenance_plan_view_form"/>
+        <field name="priority" eval="100"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='interval']" position="attributes">
+                <attribute name="attrs">{'readonly': [('meter_mode', '=', True)]}</attribute>
+            </xpath>
+            <xpath expr="//field[@name='interval_step']" position="attributes">
+                <attribute name="attrs">{'readonly': [('meter_mode', '=', True)]}</attribute>
+            </xpath>
+            <xpath expr="//field[@name='maintenance_plan_horizon']" position="attributes">
+                <attribute name="attrs">{'readonly': [('meter_mode', '=', True)]}</attribute>
+            </xpath>
+            <xpath expr="//field[@name='planning_step']" position="attributes">
+                <attribute name="attrs">{'readonly': [('meter_mode', '=', True)]}</attribute>
+            </xpath>
+            <xpath expr='//form/sheet/notebook' position="before">
+                <group>
```

Comment:
For better inheritance in the future
```suggestion
                <group name="group_meter_mode">
```

Reply:
You're right. I did the same as in the other groups of maintenance_plan. I will improve this.